feature_tests/features/producer/producer-createDocumentPointer-success.feature

```diff
@@ -160,3 +161,73 @@ Feature: Producer Create Success scenarios
       | updated_on  | NULL                                    |
       | document    | <document>                              |
       | created_on  | <timestamp>                             |
+
+  Scenario: Successfully create a Document Pointer for one directional data sync when date is provided
+    Given Producer "BaRS (EMIS)" (Organisation ID "V4T0L.YGMMC") is requesting to create Document Pointers
+    And Producer "BaRS (EMIS)" is registered in the system for application "DataShare" (ID "z00z-y11y-x22x") with pointer types
+      | system                 | value           |
+      | http://snomed.info/sct | 861421000000109 |
+    And Producer "BaRS (EMIS)" is one directional data sync
```

Comment:
Can we have a different producer name apart from BaRS(EMIS) ? maybe OneDirectionalDataSync as it will improve readability of the tests